### PR TITLE
Fix for readme with respect to the AWS DeepLearning AMI users.

### PR DESCRIPTION
**Pull request by b0noI**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ It implements the well-known encoder-decoder architecture with attention.
 
 If you are interested in collaborating or have any questions, please submit a pull request or issue.
 You can also send questions to *sockeye-dev-at-amazon-dot-com*.
+
+If you are AWS DeepLearning AMI user please refer to the corresponding section below.
```
Comment (fhieber) [inline README.md]: I think we can leave this sentence out and only mention AWS DLAMI under Installation.

File: README.md [hunk 2]
```diff
@@ -17,7 +19,16 @@ Sockeye requires:
 - numpy
 
 ## Installation
-You have two options for installing sockeye: pip and directly from source.
+
+### For AWS DeepLearning AMI users
+
+AWS DeepLearning AMI users need to run only one line in order to install the sockeye:
```
Comment (fhieber) [inline README.md]: "AWS DeepLearning AMI users only need to run the following line to install sockeye"

File: README.md [hunk 3]
```diff
@@ -17,7 +19,16 @@ Sockeye requires:
...
+
+```bash
+> sudo pip3 install sockeye --no-deps
+```
+
+For non DLAMI users - you have two options for installing sockeye: pip and directly from source.
```
Comment (fhieber) [inline README.md]: "For other environments, you can choose between installing via pip or directly from source."

File: README.md [hunk 4]
```diff
@@ -74,6 +89,8 @@ In general you can install all optional dependencies from the Sockeye source fol
 
 ### Running sockeye
 
+*AWS DeepLearning AMI user need to use python3 command instead of the python*
```
Comment (fhieber) [inline README.md]: I would move this line after the
```bash
> python -m sockeye.train <args>
```
block (line 84/101)

File: README.md (hunk 1 above)
Comment (b0noI) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (b0noI) [inline README.md]: done

File: README.md (hunk 3 above)
Comment (b0noI) [inline README.md]: done

File: README.md (hunk 4 above)
Comment (b0noI) [inline README.md]: done

**Pull request merged**